src/components/templates/Auditorium/components/Section/Section.scss

```diff
@@ -0,0 +1,105 @@
+@import "scss/constants";
+
+// If you change this, make sure to also change it in constants.ts
+$seat-size: 4vh;
+$seat-size-min: 25px;
+$seat-spacing: 3px;
+
+$spacing: calc(1 * min(var(--seat-size), var(--seat-size-min)));
+
+.section {
+  --seat-size: #{$seat-size};
+  --seat-size-min: #{$seat-size-min};
+  --seat-spacing: #{$seat-spacing};
+
+  background-attachment: fixed;
+  background-position: center 46%;
+
+  position: relative;
+  overflow: auto;
+  height: 100%;
+  padding: $spacing;
+  min-width: fit-content;
+
+  &__video-overlay {
+    position: absolute;
+
+    width: 100%;
+    height: 100%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    pointer-events: none;
+  }
+
+  &__video {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    pointer-events: all;
+
+    width: 100%;
+    height: 100%;
+    border: 1px black;
+    border-radius: 15px;
+    box-shadow: 2px 2px 20px $white;
+  }
+
+  &__seats {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: space-between;
+    position: relative;
+    width: 100%;
+    height: fit-content;
+    overflow: auto;
+  }
+
+  &__seats-row {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__seat,
+  &__empty-circle,
+  &__user-avatar {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: var(--seat-size);
+    height: var(--seat-size);
+    min-width: var(--seat-size-min);
+    min-height: var(--seat-size-min);
+
+    font-size: calc(0.75 * min(var(--seat-size), var(--seat-size-min)));
+
+    border-radius: 100%;
+    margin: $seat-spacing;
+
+    color: $black;
+    user-select: none;
+  }
+
+  &__seat {
+    background-color: rgba($white, 0.2);
+
+    &:hover {
+      background-color: rgba($white, 0.4);
+      cursor: pointer;
+    }
+
+    &:active {
+      background-color: rgba($white, 0.6);
+      cursor: pointer;
+    }
+  }
+
+  &__empty-circle {
+    background-color: rgba($black, 0);
+  }
```

Comment:
Why `empty-circle` here instead of `empty-seat` as it used to be?

Reply:
`empty-seat` implies that it is a seat that can be taken.

`empty-circle` is the elements behind the video, which are not seats actually, since they can not be taken

Reply:
Fair enough, makes sense.